{"uuid":"24908ce8-4c42-479f-aa5c-8eac43daeb36","name":"Scenario: User enters Invalid Python code and Presented with Error Message","children":["43c5ee42-37c4-48c2-8dad-942e00ddbabf"],"befores":[],"afters":[{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681592864948,"stop":1681592864949},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681592865503,"stop":1681592865504},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681592865641,"stop":1681592865641},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681592865723,"stop":1681592865723},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681592865746,"stop":1681592865749}],"start":1681592861953,"stop":1681592865750}